#!/usr/bin/env python

PACKAGE = "openai_test"  
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# Define dynamic parameters
gen.add("v0", double_t, 0, "velocity", 0.3, 0.0, 2.0)
gen.add("w0", double_t, 0, "angular velocity", 0.0, -1.0, 1.0)
gen.add("safe_distance", double_t, 0, "Safe distance to detect obstacles", 0.5, 0.1, 2.0)

exit(gen.generate(PACKAGE, "openai_test", "RobotConfig"))
